docs/dev-guides/11-blender-import.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: importing-from-blender
-title: Importing content from Blender
+title: Import From Blender
```

Comment:
The main reason you would need this is if you are adding physics. Otherwise we support plain GLB's without XRPK.

But this doc doesn't talk about physics so this entire page isn't really helpful.

The entire part about the web interface is also outdated.

Reply:
3a12f86